DataStructures/DoublyLinkedList/DoublyLinkedList.cs

```diff
@@ -212,85 +213,92 @@ public DoublyLinkedListNode<T> GetAt(int position)
             var current = Head;
             for (var i = 0; i < position; i++)
             {
-                current = current.Next;
+                current = current!.Next;
             }
 
-            return current;
+            return current ?? throw new ArgumentOutOfRangeException();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a constructor overloads that allows a more meaningful exception message to be provided.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=4735456)